resources/qml/TimelineView.qml

```diff
@@ -26,13 +26,15 @@ Page {
 		category: "user"
 		property bool avatar_circles: true
 		property string emoji_font_family: "default"
+		property double font_size: 0
```

Comment:
Please use the same default as in the Usersettings.cpp (which is the system font size default)